app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parameters = command.split(" ")
```

Comment:
No need to pass `" "` inside split, because it is a default separator

Reply:
@MaksNochvai not fixed

Reply:
Also before getting by index element you must validate that command have 3 parameters